tests/dummy/app/templates/application.hbs

```diff
@@ -6,4 +6,9 @@
   </div>
 {{/code-block}}
 
+<h3>With `line-numbers` plugin:</h3>
+{{#code-block class="line-numbers"}}&lt;div&gt;
```

Comment:
I think you can relax a bit and use the class syntax for now. When more people use this addon you'll receive feedbacks and suggestions to improve the syntax in the future.